grakn-dashboard/src/components/graphPage/footer/nodeLabelPanel.vue

```diff
@@ -38,6 +38,7 @@ along with Grakn. If not, see <http://www.gnu.org/licenses/gpl.txt>.
             <div class="side-column"><colour-picker :node="node"></colour-picker></div>
         </div>
         <div class="panel-footer">
+            <button class="btn reset" @click="configureNode()">Reset</button>
```

Comment:
Maybe this button should be "Reset Label" to make it more explicit? Not sure, but it feels less ambiguous

Reply:
If we keep the functionalities for resting the colours and labels separate then you are right.